# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1481915617 18000
#      Fri Dec 16 14:13:37 2016 -0500
# Node ID eb064b1b4269f1aab6fb43eb0c382fa60316ef6d
# Parent  0b5d9978d7b1d386fdd3738b0a78211d2e5859da
* __ode15__.cc: Fix botched style fix.

diff --git a/libinterp/dldfcn/__ode15__.cc b/libinterp/dldfcn/__ode15__.cc
--- a/libinterp/dldfcn/__ode15__.cc
+++ b/libinterp/dldfcn/__ode15__.cc
@@ -884,17 +884,17 @@ namespace octave
   ColumnVector
   ida_user_function (const ColumnVector& x, const ColumnVector& xdot,
                      double t, octave_function *ida_fc)
   {
     octave_value_list tmp;
 
     try
       {
-        tmp = ida_fc->do_multi_index_op (1, ovl (xdot, x, t));
+        tmp = ida_fc->do_multi_index_op (1, ovl (t, x, xdot));
       }
     catch (octave::execution_exception& e)
       {
         err_user_supplied_eval (e, "__ode15__");
       }
 
     return tmp(0).vector_value ();
   }
